src/quo2/components/selectors/disclaimer/view.cljs

```diff
@@ -10,8 +10,9 @@
    {:style (merge container-style (style/container blur?))}
    [selectors/checkbox
     {:accessibility-label accessibility-label
-     :on-change           on-change
-     :checked?            checked?}]
+     :blur?               blur?
```

Comment:
Added blur property to checkbox selector, so that the disclaimer's checkbox can look blurred